tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/CommonLanguageHelpers.cs

```diff
@@ -0,0 +1,278 @@
+using System.ComponentModel;
+using System.Threading;
+using Azure.Sdk.Tools.Cli.Configuration;
+using Azure.Sdk.Tools.Cli.Helpers;
+using Azure.Sdk.Tools.Cli.Microagents;
+using Azure.Sdk.Tools.Cli.Microagents.Tools;
+using Azure.Sdk.Tools.Cli.Models.Responses.Package;
+using Azure.Sdk.Tools.Cli.Prompts.Templates;
+
+namespace Azure.Sdk.Tools.Cli.Services;
+
+/// <summary>
+/// Interface for common helper methods for validation checks
+/// </summary>
+public interface ICommonValidationHelpers
+{
+    /// <summary>
+    /// Common changelog validation implementation
+    /// </summary>
+    /// <param name="packageName">SDK package name (provided by language-specific implementation)</param>
+    /// <param name="packagePath">Absolute path to the package directory</param>
+    /// <param name="fixCheckErrors">Whether to attempt to automatically fix changelog issues</param>
+    /// <param name="ct">Cancellation token</param>
+    /// <returns>CLI check response containing success/failure status and response message</returns>
+    Task<PackageCheckResponse> ValidateChangelogCommon(
```

Comment:
Can you please remove the suffix Common? Is there a language specific check for the same functionality?